content/administration/on_premise.rst

```diff
@@ -7,159 +7,110 @@ On-premise
+  If that is the case, manually change the database(s) UUID or `send a support ticket
+  <https://www.odoo.com/help>`_.
 
-Once your database has the correct number of users, the expiration message
-will disappear automatically after a few days, when the next verification occurs.
-We understand that it can be a bit frightening to see the countdown,
-so you can :ref:`force an Update Notification <force_ping>`  to make the message disappear
-right away.
+- As the update notification must be able to reach Odoo's subscription validation servers, ensure
+  your **network and firewall settings** allow the Odoo server to open outgoing connections
+  towards:
 
-Database expired error message
-------------------------------
+  - `services.odoo.com` on port `443` (or `80`)
+  - for older deployments, `services.openerp.com` on port `443` (or `80`)
 
-If your database reaches its expiration date before your renew your subscription,
-you will encounter this message:
+  These ports must be kept open even after registering a database, as the update notification runs
+  once a week.
 
-.. image:: on_premise/database_expired.png
-   :alt: This database has expired.
+Too many users error
+--------------------
 
-This **blocking** message appears after a non-blocking message that lasts 30 days.
-If you fail to take action before the end of the countdown, the database is expired.
+If you have more users in a local database than provisioned in your Odoo Enterprise subscription,
+the following message should be displayed.
 
-Solutions
-~~~~~~~~~
+.. image:: on_premise/add-more-users.png
+   :alt: Too many users on a database error message
 
-* Renew your subscription: follow the link and renew your subscription - note that
-    if you wish to pay by Wire Transfer, your subscription will effectively be renewed
-    only when the payment arrives, which can take a few days. Credit card payments are
-    processed immediately.
-* Contact our `Support <https://www.odoo.com/help>`__
+When the message appears, you have 30 days to act before the database expires. The countdown is
+updated every day.
 
-None of those solutions worked for you? Please contact our
-`Support <https://www.odoo.com/help>`__
+To resolve the issue, either:
 
-.. _force_ping:
+- **Add more users** to your subscription by clicking the :guilabel:`Upgrade your subscription` link
+  displayed in the message to validate the upsell quotation and pay for the extra users.
+- :ref:`Deactivate users <users/deactivate>` and **reject** the upsell quotation.
 
-.. _duplicate_premise:
+.. important::
+   If the subscription is on a monthly subscription plan, the database will automatically update to
+   reflect the added user. If it is on a yearly or multi-year subscription, an expiration banner
+   will appear in the database. The Odoo customer can create the upsell quote themselves by clicking
+   the banner to update the subscription or a Customer Service Manager can help resolve the issue.
```

Comment:
```suggestion
   If you are on a monthly subscription plan, the database will automatically update to reflect the
   added user(s). If you are on a yearly or multi-year plan, an expiration banner will appear in the
   database. You can create the upsell quotation by clicking the banner to update the subscription
   or `send a support ticket <https://www.odoo.com/help>`_ to resolve the issue.
```
Suggestion to keep the style consistent. Not using you/your (except in headings) isn't part of the official guidelines, so we don't enforce it.